src/plugins/webpack/index.ts

```diff
@@ -70,15 +70,26 @@ const findWebpackDependencies: GenericPluginCallback = async (configFilePath, {
       const dependencies = (config.module?.rules?.flatMap(resolveRuleSetDependencies) ?? []).map(loader =>
         loader.replace(/\?.*/, '')
       );
-      const entries = (
-        cfg.entry
-          ? typeof cfg.entry === 'string'
-            ? [cfg.entry]
-            : Array.isArray(cfg.entry)
-            ? cfg.entry
-            : Object.values(cfg.entry).map(entry => (typeof entry === 'string' ? entry : entry.filename))
-          : []
-      ).map(entry => (config.context ? join(config.context, entry) : entry));
+      let entries: string[] = [];
```

Comment:
Let's stay consistent, I prefer `const` over `let` where possible.

Reply:
I do too but I can't make it `const` and also reassign it further down.

Reply:
Then please make the re-assignment a new `const`

Reply:
I was just going to suggest, how about this? https://github.com/thomasjm/knip/commit/cb190223a338ad51615783d402d8d32f03031f4a

Reply:
That's fine as well

Reply:
Ok done